### DISCUSSION TITLE
opcua-asyncio/examples/simple-client-server-xml/

### DISCUSSION THREAD

**Discussion by FatimaRani** [FreeOpcUa/opcua-asyncio#794] [Q&A]
cant even run this simple example

getting error

C:\Users\Fatima\PycharmProjects\pythonOPCUA\venv\Scripts\python.exe C:/Users/Fatima/PycharmProjects/pythonOPCUA/server-xml.py
C:\Users\Fatima\PycharmProjects\pythonOPCUA\server-xml.py:53: RuntimeWarning: coroutine 'Server.import_xml' was never awaited
  self.server.import_xml(model_filepath)
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
Traceback (most recent call last):
  File "C:\Users\Fatima\PycharmProjects\pythonOPCUA\server-xml.py", line 82, in <module>
    with HelloServer(
  File "C:\Users\Fatima\PycharmProjects\pythonOPCUA\server-xml.py", line 62, in __init__
    hellower_say_hello = hellower.get_child("0:SayHello")
AttributeError: 'coroutine' object has no attribute 'get_child'
sys:1: RuntimeWarning: coroutine 'Server.get_namespace_index' was never awaited
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
sys:1: RuntimeWarning: coroutine 'Node.get_child' was never awaited
RuntimeWarning: Enable tracemalloc to get the object allocation traceback

**Comment by AndreasHeine** [comment]
looks like a missing await in front of the get_child() method!
